content/en/getting_started/tracing/_index.md

```diff
@@ -20,159 +20,197 @@ further_reading:
 
-```text
-vagrant init ubuntu/jammy64
-vagrant up
-vagrant ssh
-```
+1. [Create a Datadog account][1] if you haven't already.
+1. Find or create a [Datadog API key][2].
+1. Start up a Linux host or VM.
 
-To install the Datadog Agent on a host, use the [one line install command][6] updated with your [Datadog API key][7]:
+## Set up Datadog APM
 
-```shell
-DD_API_KEY=<DATADOG_API_KEY> DD_SITE="{{< region-param key="dd_site" >}}" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
-```
+To set up Datadog APM without needing to touch your application code, use Single Step APM Instrumentation:
 
-### Validation
+1. Run the installation command:
 
-Verify the Agent is running with the [status command][8]:
+   ```shell
+    DD_API_KEY=<YOUR_DD_API_KEY> DD_SITE="<YOUR_DD_SITE>" DD_APM_INSTRUMENTATION_ENABLED=host DD_ENV=<AGENT_ENV> bash -c "$(curl -L https://s3. amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
+    ```
+ 
+    Replace `<YOUR_DD_API_KEY>` with your [Datadog API key][4], `<YOUR_DD_SITE>` with your [Datadog site][3], and `<AGENT_ENV>` with the environment your Agent is installed on (for example, `development`).
 
-```shell
-sudo datadog-agent status
-```
+1. Start a new shell session.
+1. Restart the services on your host or VM.
+1. Verify the Agent is running:
 
-After a few minutes, verify the Agent is connected to your account by checking the [Infrastructure List][9] in Datadog.
+    ```shell
+   sudo datadog-agent status
+   ```
 
-## Datadog APM
+This approach automatically installs the Datadog Agent, enables Datadog APM, and [instruments][5] your application.
 
-### Follow the in-app documentation (recommended)
+## Create an application
 
-For the remaining steps, follow the [Quick start instructions][10] within the Datadog site for the best experience, including:
+To create an application to observe in Datadog:
 
-- Step-by-step instructions scoped to your deployment configuration (in this case, a host-based deployment).
-- Dynamically set `service`, `env`, and `version` tags.
-- Enable the Continuous Profiler, ingesting 100% of traces, and Trace ID injection into logs during setup.
+1. On your Linux host or VM, create a new Python application named `hello.py`. For example, `nano hello.py`.
+1. Add the following code to `hello.py`:
 
+    {{< code-block lang="python" filename="hello.py" collapsible="true" disable_copy="false" >}}
+  from flask import Flask
+  import random
 
-### Enable APM
+  app = Flask(__name__)
+  
+  quotes = [
+      "Strive not to be a success, but rather to be of value. - Albert Einstein",
+      "Believe you can and you're halfway there. - Theodore Roosevelt",
+      "The future belongs to those who believe in the beauty of their dreams. - Eleanor Roosevelt"
+  ]
+  
+  @app.route('/')
+  def index():
+      quote = random.choice(quotes)+"\n"
+      return quote
+  
+  if __name__ == '__main__':
+      app.run(host='0.0.0.0', port=5050)
+      ```
```

Comment:
I believe these tick marks shouldn't be there? Typo?

Reply:
Good catch!